wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement for the _current version_ of the software available on this website. If there is a conflict between the English language version and any translation, the English version shall apply. Earlier versions may have had different privacy terms that are no longer applicable or supported. Users are encouraged to use the most current version of the Jamulus software.**
```

Comment:
```suggestion
**The English version of this document represents the official privacy statement for the _current version_ of the software available on this website. Earlier versions of the software and non-English translations of this privacy policy may have different terms that are no longer applicable or supported. Users are encouraged to use the most current version of the Jamulus software.**
```

Reply:
That last sentence seems a little out of place. Do we say this anywhere else? That _Users are encouraged to use the most current version of the Jamulus software_? I saw distros recently with many old versions of Jamulus. Are we saying what we encourage here due to the privacy policy? It's generally a best practice to use latest bits. But why are we mentioning it here?

Reply:
I'm not sure what "different terms" refers to here exactly (perhaps outdated vocabulary, but also in translated cases things might just be out of date). But it seems like a sort of quasi-legalese reminder that we may make updates to Jamulus that could affect issues mentioned in the privacy policy. Seems fair enough.

Reply:
I've removed the sentence from my suggestion.

Reply:
@gilgongo please apply this if you're ok with it.